activesupport/lib/active_support/dependencies.rb

```diff
@@ -770,27 +745,8 @@ def remove_constant(const) #:nodoc:
       begin
         parent.instance_eval { remove_const to_remove }
       rescue NameError
-        log "the constant #{const} is not reachable anymore, skipping"
```

Comment:
The removal of `#log` here is a bit weird because the `rescue` does nothing anymore...